.github/workflows/ci.yml

```
@@ -120,7 +120,7 @@ jobs:
         run: mvn -T 1.5C -B clean verify --settings .github/settings.xml
 
       - name: Publish
-        run: mvn -T 1.5C -B deploy --settings .github/settings.xml
+        run: mvn -T 1.5C -B deploy --settings .github/settings.xml -DaltDeploymentRepository=github-set::https://maven.pkg.github.com/${{ github.repository }}
```

```suggestion
        run: mvn -T 1.5C -B deploy --settings .github/settings.xml -DaltDeploymentRepository=set-github::https://maven.pkg.github.com/${{ github.repository }}
```
Here should be _set-github_ right? In parent pom by <distributionManagement> is _set-github_